code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -341,7 +341,7 @@
 	var/datum/antagonist/heretic/cultie = owner.has_antag_datum(/datum/antagonist/heretic)
 	if(!cultie)
 		return FALSE
-	return LAZYLEN(folloers)>=target_amount
+	return LAZYLEN(cultie.folloers)>=target_amount
```

Comment:
folloers? well if thats how you code followers ok.

Reply:
Based moment

Reply:
That should have been fixed...